src/haddock/modules/topology/topocg/aa2cg.py

```diff
@@ -201,18 +202,19 @@ def center_of_mass(entity, geometric=False):
     # If there is a single atom with undefined mass complain loudly.
     if "ukn" in set(masses) and not geometric:
         raise ValueError("Some Atoms don't have an element assigned.\n"
```

Comment:
Use `os.linesep` instead of `\n`, for OS interoperability.
```suggestion
        raise ValueError(f"Some Atoms don't have an element assigned.{os.linesep}"
```